documentation/en/api-v0-methods-miner.md

```diff
@@ -465,7 +465,9 @@ Inputs:
     {
       "SealProof": 8,
       "SectorNumber": 9,
-      "SectorKey": null,
```

Comment:
This were a side effect of adding a docs example for the `*cid.Cid` type. I'm inclined to keep them even though the CID prefixes may be wrong in some cases. If someone complains, we might have to add CID newtypes to attach specific CID examples to them.